custom-words.txt

```diff
@@ -3901,4 +3901,6 @@ capacityreservationgroup
 capacityreservationgroups
 ciam
 CIAM
-msvideo
\ No newline at end of file
+VTPM
```

Comment:
We are trying to move away from custom-workds.txt can you instead following the instructions which I'm updating in https://github.com/Azure/azure-rest-api-specs/pull/26874 to add these to the cspell config file instead.